main.go

```diff
@@ -123,7 +140,7 @@ func main() {
 		NewDatasetRoutingEnabled:     cfg.NewDatasetRoutingEnabled,
 		PrefixDatasetHandler:         prefixDatasetHandler,
 		DatasetClient:                datasetClient,
-		HealthCheckHandler:           hc.Handler,
+		// HealthCheckHandler:           hc.Handler,
```

Comment:
Is there any reason the health check handler has been removed or will this be uncommented when ready?

Reply:
my fault. I commented it out to reduce log noise. It's back in locally but I haven't pushed the change yet.